sale_channel_hook_stock_variation/models/stock_move.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+from odoo.addons.queue_job.job import identity_exact
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _notify_stock_variation(self):
+        for rec in self:
+            products_moved = rec.move_line_ids.mapped("product_id")
+            for product in products_moved:
+                product.with_delay(identity_key=identity_exact)._notify_stock_variation(
+                    rec.warehouse_id
```

Comment:
we do not ware of the warehouse here

Reply:
done
If I understand correctly any stock update should trigger binding notifications for all warehouses